plugins/backstage-plugin-coder/src/plugin.ts

```diff
@@ -191,8 +191,14 @@ export { useWorkspacesCardContext } from './components/CoderWorkspacesCard/Root'
  */
 export { useCoderWorkspacesConfig } from './hooks/useCoderWorkspacesConfig';
 export { useCoderSdk } from './hooks/useCoderSdk';
+export { useCoderQuery } from './hooks/reactQueryWrappers';
```

Comment:
Appears to be unused.

Reply:
Oh, all of these are the end-user exports. It's not used because it's getting passed directly to the user

Reply:
Oof I completely misread `export` and `import` lol